### ISSUE TITLE
Compiller error

### ISSUE THREAD

**Issue by Warrior109** [crystal-lang/crystal#6524]
After I compile my program, an error has occurred:
![image](https://user-images.githubusercontent.com/20104771/43989284-d10eab70-9d4f-11e8-92a8-705a4e10316c.png)
![image](https://user-images.githubusercontent.com/20104771/43989287-da996400-9d4f-11e8-8e87-26cc398260c6.png)
Source code for project here: https://github.com/Warrior109/crsfml_tutorial

**Comment by Warrior109** [comment]
Problem in `struct SnakeTutorial::EventManager::EventType` in `EVENTS` constant

**Comment by straight-shoota** [comment]
Could you please copy the program output directly into the issue as text? Thanks!

**Comment by Warrior109** [comment]
```ubuntu@ip-172-31-21-114:~/programing/crystal/snake_tutorial(master)$ crystal src/snake_tutorial.cr
BUG: trying to assign SnakeTutorial::EventManager::EventType+:Class <- (SnakeTutorial::EventManager::EventType::Closed:Class | SnakeTutorial::EventManager::EventType::GainedFocus:Class | SnakeTutorial::EventManager::EventType::Joystick:Class | SnakeTutorial::EventManager::EventType::KeyDown:Class | SnakeTutorial::EventManager::EventType::KeyUp:Class | SnakeTutorial::EventManager::EventType::Keyboard:Class | SnakeTutorial::EventManager::EventType::LostFocus:Class | SnakeTutorial::EventManager::EventType::MButtonDown:Class | SnakeTutorial::EventManager::EventType::MButtonUp:Class | SnakeTutorial::EventManager::EventType::Mouse:Class | SnakeTutorial::EventManager::EventType::MouseEntered:Class | SnakeTutorial::EventManager::EventType::MouseLeft:Class | SnakeTutorial::EventManager::EventType::MouseWheel:Class | SnakeTutorial::EventManager::EventType::TextEntered:Class | SnakeTutorial::EventManager::EventType::WindowResized:Class) (Exception)
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
  from ???
Error: you've found a bug in the Crystal compiler. Please open an issue, including source code that will allow us to reproduce the bug: https://github.com/crystal-lang/crystal/issues
```

**Comment by Warrior109** [comment]
Already fixed in crystal 0.26.0
I think with this changes: https://github.com/crystal-lang/crystal/pull/6351
